### DISCUSSION TITLE
Static Resource PrimaryXXX

### DISCUSSION THREAD

**Discussion by mohsen-id** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3536] [Q&A] [answered]
I migrated to version 5 and in some parts of the code I used Static Resource **PrimaryXXX** colors that do not exist now. If I add `MaterialDesignColor.XXX.xaml` as a `ResourceDictionary`, it affects BundledTheme and changes the Foreground color for example. 

What is the best way to access these PrimaryXXX colors in version 5?

my App.xaml:
`<ResourceDictionary.MergedDictionaries>
          <md:BundledTheme BaseTheme="Light" PrimaryColor="DeepOrange" SecondaryColor="Lime" />
          <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesign3.Defaults.xaml" />
  </ResourceDictionary.MergedDictionaries>`

**Comment by Keboo** [comment, marked as answer]
Linking the answer in the other discussion chain for anyone who comes across this in the future. 
https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/discussions/3466#discussioncomment-9220247

https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/discussions/3466#discussioncomment-9247170
